app-emulation/qemu/qemu-7.2.4-r1.ebuild

```diff
@@ -273,6 +273,7 @@ BDEPEND="
 	$(python_gen_impl_dep)
 	dev-lang/perl
 	dev-build/meson
+	dev-build/ninja
```

Comment:
Shouldn't this be `app-alternatives/ninja`?

Reply:
Done